### ISSUE TITLE
CREATE INDEX doesn't automatically bound index to the schema property in v3.0.0

### ISSUE THREAD

**Issue by dastoori** [orientechnologies/orientdb#8268]
## Expected behavior  

`CREATE INDEX` should automatically bound index to a property using index name:

```sql
CREATE INDEX User.id UNIQUE    /* throws error */ 

CREATE INDEX User.id ON User (id) UNIQUE    /* working */ 
```

## Actual behavior  

It throws this error:

```
com.orientechnologies.orient.core.exception.ODatabaseException: Impossible to create an index without specify the key type or the associated property: CREATE INDEX User.id UNIQUE 
```

## Steps to reproduce  

Run these commands (from https://orientdb.com/docs/3.0.x/sql/SQL-Create-Index.html):

```sql
CREATE PROPERTY User.id BINARY
CREATE INDEX User.id UNIQUE
```

**Comment by luigidellaquila** [comment]
Hi @dastoori 

That syntax will be deprecated soon, it's there only for backward compatibility in the legacy SQL executor. Please use the full syntax.
Anyway, I'll give it a look, probably it's an easy fix

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @dastoori 

I just pushed a fix for this, it will be released with v 3.0.2

Thanks

Luigi
